tests/stress/km/stress_tests_km.cpp

```diff
@@ -1060,3 +1198,29 @@ TEST_CASE("native_invoke_program_restart_extension_v4_test", "[mt_stress_test]")
     local_test_control_info.extension_restart_enabled = true;
     _mt_invoke_prog_stress_test(EBPF_EXECUTION_NATIVE, local_test_control_info);
 }
+
+TEST_CASE("sockaddr_invoke_program_test", "[mt_stress_test]")
+{
+    // Test layout:
+    // 1. Load the "cgroup_mt_connect6.sys" native ebpf program.
+    //    - This program monitors an IPv6 endpoint, ::1:<target_port>. On every invocation, the program returns a
```

Comment:
```suggestion
    //    - This program monitors an IPv6 endpoint, [::1]:<target_port>. On every invocation, the program returns a
```

Reply:
when using `<address>:<port>` syntax, IPv6 uses `[]` around the address

Reply:
ah. ofc that makes sense.  fixed.